examples/api/lib/widgets/tween_animation_builder/tween_animation_builder.0.dart

```diff
@@ -11,42 +11,53 @@ void main() => runApp(const TweenAnimationBuilderExampleApp());
 class TweenAnimationBuilderExampleApp extends StatelessWidget {
   const TweenAnimationBuilderExampleApp({super.key});
 
+  static const Duration duration = Duration(seconds: 1);
+
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
       home: Scaffold(
-        appBar: AppBar(title: const Text('TweenAnimationBuilder Sample')),
+        appBar: AppBar(
+          title: const Text('TweenAnimationBuilder Sample'),
+        ),
         body: const Center(
-          child: TweenAnimationBuilderExample(),
+          child: TweenAnimationBuilderExample(duration: duration),
         ),
       ),
     );
   }
 }
 
 class TweenAnimationBuilderExample extends StatefulWidget {
-  const TweenAnimationBuilderExample({super.key});
+  const TweenAnimationBuilderExample({
+    required this.duration,
```

Comment:
Maybe make this optional, and provide a default duration of 1 second?

Reply:
@gspencergoog thanks for the review!
In the recent commit I updated the duration to be optional and provided a default value. 
Could you please take a look again?